pkg/cli/mirror/mirror.go

```diff
@@ -35,6 +35,53 @@ import (
 	"github.com/openshift/oc-mirror/pkg/metadata/storage"
 )
 
+var (
+	mirrorlongDesc = templates.LongDesc(
+		` 
+		Create and publish user-configured mirrors with a declarative configuration input.
+		Accepts an argument defining the destination for the mirrored images using the prefix file:// for a local mirror packed into a 
+		tar archive or docker:// for images to be stream registry to registry without being stored locally. The default docker credentials are 
+		used for authenticating to the registries. The podman location for credentials is also supported as a secondary location.
+
+		When using file mirroring, the --from and --config flags control the location of the images to mirror. The --config flag accepts
+		an imageset configuration file and the --from flag accepts the location of the imageset on disk. The --from input can be passed as a 
+		file or directory, but must contain only one image sequence. The naming convention for an imageset is mirror_seq<sequence number>_<tar count>.tar.
+
+		The location of the workspace used defaults to oc-mirror-workspace in the current directory when publishing image content or using the  to 
```

Comment:
The second part of this sentence after "or" looks incomplete. Not sure what this should be

Reply:
Thanks! I think this needs a `when` before using.